pkg/featuregate/featuregate.go

```diff
@@ -60,6 +60,14 @@ var (
 		featuregate.WithRegisterDescription("enables features associated to the Prometheus Operator"),
 		featuregate.WithRegisterFromVersion("v0.82.0"),
 	)
+
+	// EnableCRMetrics is the feature gate that enables metrics about the collectors managed by this operator.
+	EnableCRMetrics = featuregate.GlobalRegistry().MustRegister(
```

Comment:
@swiatekm-sumo IIRC we had a discussion about not using a feature gate for use-case like this (a feature that might never be enabled by default).